clients/java-logger/library/src/test/java/com/abcxyz/lumberjack/auditlogclient/processor/JustificationProcessorTests.java

```diff
@@ -78,14 +93,31 @@ public void processShouldPopulateJustificationMetadata() throws Exception {
                         ListValue.newBuilder()
                             .addValues(Value.newBuilder().setStringValue("test-aud")))
                     .build())
+            .putFields(
+                "justs",
+                Value.newBuilder()
+                    .setListValue(
+                        ListValue.newBuilder()
+                            .addValues(Value.newBuilder().setStructValue(wantJustificationsClaim)))
+                    .build())
             .build();
     Struct wantMetadata =
         Struct.newBuilder()
             .putFields(
                 "justification", Value.newBuilder().setStructValue(wantJustification).build())
             .build();
+    RequestMetadata wantRequestMetadata =
+        RequestMetadata.newBuilder()
+            .setRequestAttributes(
+                Request.newBuilder()
+                    .setReason("[{\"category\":\"explanation\",\"value\":\"need-access\"}]")
```

Comment:
@capri-xiyue Can you confirm on the Go side we're handling it as a list as well?

Reply:
Checked JVS java client, it is also using the [justs claim as a list](https://github.com/abcxyz/jvs/blob/e718d4664467b880991b8e2a400070c2aa93a0b9/client-lib/java/src/main/java/com/abcxyz/jvs/JvsClient.java#L76). Also tried to convert the justs Claim directly to json using both gson and jackson, neither of them worked because Claim holds the value in a generic way.